skyvern/webeye/browser_factory.py

```
@@ -73,53 +134,94 @@ async def listen_to_download(download: Download) -> None:
+                                            filename=value,
+                                        )
+                                        found_in_query = True
+                                        break
+                                if not found_in_query:
+                                    LOG.info(
+                                        "Add extension according to download url path",
+                                        workflow_run_id=workflow_run_id,
+                                        task_id=task_id,
+                                        filepath=str(file_path) + extracted_extension,
+                                    )
+                            except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)